### ISSUE TITLE
Solid.js support

### ISSUE THREAD

**Issue by aboodman** [rocicorp/replicache#1014]
Right now it is possible to use Replicache with Solid, but not easy since there are no samples and not efficient since Replicache's `subscribe()` method returns the entire dataset each time it runs defeating Solid's incremental re-rendering.

We should have a sample app / quickstart for Solid and do whatever work internally necessary to make it work well. This includes at least #1010 .

**Comment by thdxr** [comment]
+1

**Comment by m3rashid** [comment]
Hey, any update on this ?

**Comment by aboodman** [comment]
No, we've shifted our main focus to Zero (which does have Solid support incidentally): https://zero.rocicorp.dev/. We're supporting Replicache but not new features.
